pkgs/development/node-packages/node-packages.json

```diff
@@ -3,7 +3,6 @@
 , "@antora/cli"
 , "@antora/site-generator-default"
 , "@bitwarden/cli"
-, "@vue/cli"
```

Comment:
i am adding those, right now. Don't know how they went missing.

Reply:
Ok, done in https://github.com/NixOS/nixpkgs/pull/89184/commits/6149df8fd095cb6fe5a0a4fc2438e2805942ad59